AdminUi/src/AdminUi/Controllers/IdentitiesController.cs

```diff
@@ -90,6 +95,20 @@ public async Task<IActionResult> GetIdentityByAddress([FromRoute] string address
 
         return Ok(response);
     }
+
+    [HttpGet("Overview")]
+    [ProducesResponseType(typeof(PagedHttpResponseEnvelope<IdentityOverviewDTO>), StatusCodes.Status200OK)]
+    [ProducesError(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> GetIdentitiesOverview([FromQuery] PaginationFilter paginationFilter, CancellationToken cancellationToken)
```

Comment:
Hmm, is this a temporary solution? Because the goal is that the Overviews can be obtained via the `GET /Identities` route.